docs/source/AdministratorGuide/Introduction/configurationbasics.rst

```diff
@@ -23,13 +25,15 @@ Same can be said for executors: you can have many residing on different hosts.
 The same can't be said for agents. Some of them can be duplicated, BUT require a proper configuration,
 and for this you need to read further in the guide (See `scalingLimitations`).
 
+System API endpoints can also be duplicated on different hosts, but to have a single entry point it is desirable to run it on one host.
```

Comment:
I don't think this can be a suggestion. The medium term plan is to run services on k8s.

Reply:
removed